src/fat.ml

```diff
@@ -538,6 +538,18 @@ module KV_RO(B: Mirage_block.S) = struct
       | Error e -> Error (`FS e)
       | Ok l -> Ok Cstruct.(to_string (concat l))
 
+  let get_partial t key ~offset ~length =
+    FS.read t (Mirage_kv.Key.to_string key) offset length >|= function
+    | Error e -> Error (`FS e)
```

Comment:
`FS.read` returns \`Is_a_directory if key is a directory, but `get_partial` should returns \`Value_expected. Is that ok to use something inspired by `get` (and change get to use `get_partial` as length is bound checked against the file real size in `read_file`) ?